lib/govuk_docker_cli.rb

```diff
@@ -28,4 +29,10 @@ def prune
   def run_this(*args)
     Commands::RunThis.new(options[:stack], args).call
   end
+
+  desc "doctor", "Various tests to help diagnose issues when running govuk-docker"
+  def doctor
```

Comment:
Would you be able to move this after `compose` so this stays in alphabetical order?

Reply:
👍 yep done